src/web/mjs/connectors/RainDropFansub.mjs

```diff
@@ -0,0 +1,15 @@
+import MangaReaderCMS from './templates/MangaReaderCMS.mjs';
+
+export default class RawMangaUpdate extends MangaReaderCMS {
```

Comment:
The classname does not match the website snd should be changed

Reply:
sry. I will make the change